src/Configuration/Programmer/Programmer_helper.cpp

```diff
@@ -241,6 +296,7 @@ std::vector<Device> extractDeviceList(const std::string& devicesString) {
         device.tapInfo.irLen = tap.irLen;
         device.tapInfo.tapName = tap.tapName;
         devices.push_back(device);
+        devices.push_back(device);
```

Comment:
Is it correct to add device twice?

Reply:
ops.. It was mistake when I did the code merged. Going to delete the extra line. Thanks for catching